README.md

```diff
@@ -24,6 +24,11 @@ $> cp sample.env .env
 $> npm start
 ```
 
+## Documentation
+
+- [Deployment](docs/Deployment.md)
```

Comment:
Honestly, I forgot they existed :flushed: 

Both options are good in that documentation is stored in git so it's available locally and supports Markdown. But, my preference for putting the documentation on the code repository rather than a separate documentation repository comes from my belief that [monolithic repositories are better than multiple repositories](http://danluu.com/monorepo/). In a real sense, putting the documentation in a separate repository makes it second-class and less likely that we'll add/update documentation. And as much as I dislike writing documentation, I know that I'll appreciate past-me writing it up when things are broken.